### PR TITLE
Increases codeside roundstart Cyborg slots to 3.

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

This PR ups the codebase defaults to three roundstart Cyborg slots.
## Why It's Good For The Game

I got a bit of GBP banked. Might as well spend a little.

[Codebase has configed in two borgs since I made the changes for Spookuni during their last term.](https://tgstation13.org/phpBB/viewtopic.php?p=668042#p668042) This adds one extra codebase-side default over the server status quo.

With new Silicon Policy in play, I think it's time to open up the roundstart cyborgs to be a full and complete faction alongside their master AI. This could potentially allow for a fully staffed silicon """department""" roundstart. This makes the silicon team more impactful right from the get-go and opens up more scope for dedicated silicon mains to actually get silicon roles in a unique and often contested jobspace.

That is definitely a buff to Malf AI and may cause fewer roundstart borgings in Robotics, so it may take away a minor bit of job content from them. However, Robos already have modsuits, circuits and mechs - plus giving them more roundstart borgs to eventually upgrade and subvert and generally play alongside could still contribute to job content overall.

It's also a buff to individual borg players, in the sense that their module selection now has less overall impact roundstart and thus they may be able to pick their module of choices more often without gimping their master AI's ability to interact with the shift.
## Changelog
:cl:
balance: There are now 3 roundstart cyborg job slots open by default.
/:cl:

### CODE REVIEW

**Comment by Spookuni** [comment]
Timber has threatened violence against my person if I don't crosspost my short screed on why borgs are good, so with discord formatting and accompanying illegibility, here goes:

Borgs are the sweet spot in terms of bringing many of the distinctions of silicon play (extreme specialisation in focus areas, required compliance with changing laws and lawsets, pseudo focus on supportive / cooperative action due to point #1, some other good stuff) in to rounds without the fundamental not-playing-the-same-game alteration that multiple AIs, or even single AIs in many cases are dragged down by.

Borgs work well in all three interaction conditions (cooperative, apathetic, and malicious) while having a major difference in core gameplay from carbons and not suffering from the AI core effect (the AI core is hard to get to, but once you're there the AI loses without counterplay to an injured one armed assistant with an emergency oxygen tank) where they lack critical opportunities for engagement from everyone else.

You can kill a borg as an antag, you can kill a borg as a crewmember, you can revive a borg killed by an antag as a crewmember, borgs fear personal-scale threats and have to make choices about whether or not to run away, fight, contain or do whatever else on a personal-scale.

TL:DR - Borgs good add more borgs

**Comment by carlarctg** [comment]
Your code failed an integration check on icebox. Admins cant code lol.

Circuits is science content not robotics content, though admittedly they have access to it, and it's irrelevant as a whole because, well, they have enough content anyways.

I think it'd be fine to have 3 robots zipping around at least conceptually, even though silicon mains don't deserve good things. 

Borgs are extremely fucking annoying to fight especially with the stupid flash change, and this roughly triples the amount of borgs around, which is concerning to me. Cyborgs should be able to be fought without flashes, it is insane that they are still simplemobs in everything but EMPs and flashes.

**Comment by GPeckman** [comment]
> That is definitely a buff to Malf AI and may cause fewer roundstart borgings in Robotics, so it may take away a minor bit of job content from them.

There were basically zero roundstart borgings after the number of slots was upped to two, so this can't really make things much worse.

From my experience playing roboticist on malf AI rounds, its only going to be a significant buff if the roboticists were completely ignoring the ectoscopic scanner thingy and not building a single positronic brain. Its insane just how many ghosts will start hammering the scanner the second that they realize that the AI is malf. It actually became something I had to consciously ignore to avoid metagaming, it was that much of a tell that the AI was malf.

**Comment by MrMelbert** [comment]
> With new Silicon Policy in play, I think it's time to open up the roundstart cyborgs to be a full and complete faction alongside their master AI

Something something "separation of codebase and server is absolute"

**Comment by SimplySmiley123** [comment]
> > That is definitely a buff to Malf AI and may cause fewer roundstart borgings in Robotics, so it may take away a minor bit of job content from them.
> 
> There were basically zero roundstart borgings after the number of slots was upped to two, so this can't really make things much worse.
> 
> From my experience playing roboticist on malf AI rounds, its only going to be a significant buff if the roboticists were completely ignoring the ectoscopic scanner thingy and not building a single positronic brain. Its insane just how many ghosts will start hammering the scanner the second that they realize that the AI is malf. It actually became something I had to consciously ignore to avoid metagaming, it was that much of a tell that the AI was malf.

Yeah, honestly.
Barely anyone besides sillicon mains ever bothers to try and get robos to build posibrains and borg shells, unless they know there's cult targetting borgs for constructs, AI is malf, or there's a traitor/infiltrator with an emag going after borgs.
That and people who wanna grief with engineering borg.
At this rate I just ignore the sniffer whenever I play roboticist. Even if AI isn't malf I honestly hate seeing more borgs then is needed.

**Comment by Dawnseer** [comment]
> At this rate I just ignore the sniffer whenever I play roboticist. Even if AI isn't malf I honestly hate seeing more borgs then is needed.

"I ignore a good third if not half of my job" 
For shame

**Comment by Jackal-boop** [comment]
I wouldn't be against bumping it up to 4

**Comment by moocowswag** [comment]
> Circuits is science content not robotics content, though admittedly they have access to it, and it's irrelevant as a whole because, well, they have enough content anyways.

Circuits mixes very well with roboticist job content since you can use them to disable/enable maximum ogredrive from inside your mech along with the movement malfunction.

Circuits also allow for the unique interaction that is chem-mechs which is a unique way to make your mech more effective in intense situations by giving yourself extra buttons to press while in the mech (like being able to slip and knock everyone away and escape)

**Comment by san7890** [comment]
remember that this can always be adjusted through the config if server operators decide they don't like silicons

**Review by ninjanomnom — APPROVED** [review]
